src/test/groovy/ee/tuleva/onboarding/mandate/application/ApplicationDtoFixture.groovy

```diff
@@ -16,7 +16,23 @@ class ApplicationDtoFixture {
       .id(123L)
       .currency("EUR")
       .sourceFundIsin("source")
-      .fundTransferExchanges([new MandateDto.MandateFundsTransferExchangeDTO("processId", 1.0, "source", "target")])
+      .fundTransferExchanges([
+        new MandateDto.MandateFundsTransferExchangeDTO("processId", 1.0, "source", "target", null)
```

Comment:
Always prefer readability above anything else in tests. For example, if it increases readability, use [named parameters](http://docs.groovy-lang.org/latest/html/documentation/#_named_parameters_2) in Groovy. So you could do something like: 
```
new MandateFundsTransferExchangeDTO(processId: "b22ba9", amount: 1.0, sourceFundIsin: "EE1312312313", targetFundIsin: "EE2323232326", targetPik: null)
```